substrafl/remote/register/register.py

```diff
@@ -83,7 +83,9 @@
 
 def iter_dockerfile_install_command(paths: typing.List[Path], python_major_minor: str) -> str:
     for path in paths:
-        yield f"COPY {path}  {path} \nRUN python{python_major_minor} -m pip install --no-cache-dir {path} \n"
+        # Do not work with path.is_dir(), surely a race condition. Should be removed in  #123
```

Comment:
I was a bit confused, I thought this comment was an instruction because it used the imperative. Is this more like what you intended?

```suggestion
        # Does not work with path.is_dir(), surely a race condition. Should be removed in #123
```